conf/application.conf

```diff
@@ -1 +1,35 @@
 # https://www.playframework.com/documentation/latest/Configuration
+play.http.secret.key=${PLAY_SECRET_KEY}
+play.http.port=9000
+# The load balancer url is just while testing, it will be removed once we have the dns sorted
+play.filters.hosts {
+    allowed = [
+        "tdr-frontend-load-balancer-intg-1439549459.eu-west-2.elb.amazonaws.com",
+        "transfer-intg.nationalarchives.gov.uk",
+        "transfer-staging.nationalarchives.gov.uk",
+        "transfer-prod.nationalarchives.gov.uk",
+        "10.0.0.4:9000",
+        "10.0.0.5:9000",
+        "10.0.0.6:9000",
+        "10.0.0.7:9000",
+        "10.0.0.8:9000",
+        "10.0.0.9:9000",
+        "10.0.0.10:9000",
+        "10.0.0.11:9000",
+        "10.0.0.12:9000",
+        "10.0.0.13:9000",
+        "10.0.0.14:9000",
+        "10.0.0.20:9000",
+        "10.0.0.21:9000",
+        "10.0.0.22:9000",
+        "10.0.0.23:9000",
+        "10.0.0.24:9000",
+        "10.0.0.25:9000",
+        "10.0.0.26:9000",
+        "10.0.0.27:9000",
+        "10.0.0.28:9000",
+        "10.0.0.29:9000",
+        "10.0.0.30:9000",
```

Comment:
What are all these IP addresses?

Reply:
IP addresses from the subnets which the task runs in. These are created by terraform and are always the same but we don't know which one it's going to get in advance so we have to whitelist all of them. It doesn't look like you can whitelist a range in Play.

Reply:
Oh, are the IP addresses used by the load balancer health checks? Could you add a comment explaining that (or whatever the real reason is, if I've misunderstood)? I don't think it's very obvious why the host isn't just the domain name.

Reply:
Actually, I still don't understand why you need to include "localhost:9000" in the deployed environments. This might be a silly question, but what calls the app on localhost except in dev?

Reply:
I didn't mean to, I've taken localhost out of the deployed environments conf files

Reply:
👍 

What do you think about adding a comment to help people understand why the list of IP addresses is there?

Reply:
I've added in a comment to the staging and intg ones.

Reply:
Cheers. 🙂 

Sorry to keep dragging this out, but from your comment it sounds like the app won't actually start if the container's IP address isn't in this let. But the [Play docs](https://www.playframework.com/documentation/2.7.x/AllowedHostsFilter) suggest that `play.filters.hosts.allowed` is used when handling a request, rather than at startup. Is the app starting up OK but being shut down by the health check because the load balancer can't access it? Or am I misunderstanding the Play docs?

Reply:
Play will start but the load balancer uses the IP to do the health checks and it's these that are failing if the IP address isn't listed, which kills the task so the app never starts up properly. I've changed the description to mention the load balancer.

Reply:
Thanks!